packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -602,6 +628,15 @@ class AzureDevOpsServerManager implements TaskIntegrationManager {
     return {error: firstError, process: result.name}
   }
 
+  async getProject(instanceId: string, projectId: string) {
```

Comment:
-1 just to keep this manager looking like other Manager types, let's return `Error | ProjectRes` and let the calling function handle that